### PR TITLE
Added Swap Algorithm examples

### PR DESCRIPTION
**Pull request by Alaz-Oz**
Added Swapping algorithm examples
- Swap1.cpp -> SwapAny algorithm
- Swap2.cpp -> SwapNum algorithm

### CODE REVIEW

**Review by 2G-Afroz — COMMENTED** [review]
In Swap2.cpp:-
Line 4: * @brief	Swaps the values of a & b. Works with numerical values of any data type ( a & b should be of same data type )
Line 23: // Before Swapping the values of variables.
Line 23: // After Swapping the values of variables.

The output should be like: 
Before Swapping:
Value of a: 4
Value of b: 8

After Swapping:
Value of a: 8
Value of b: 4

File: Algorithms/Utils/Swap/Swap1.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+	// The initial values
+	int a = 19;
+	int b = 34;
+
+	// Printing the values of variables
+	std::cout << "Value of a: " << a << " | Value of b: " << b << std::endl;
+
+
+	// Swapping the values
```
Comment (2G-Afroz) [inline Algorithms/Utils/Swap/Swap1.cpp:27]: The documentation should be consistent.
look at this line and

File: Algorithms/Utils/Swap/Swap2.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+	// The initial values
+	int a = 19;
+	int b = 34;
+
+	// Printing the values of variables
+	std::cout << "Before Swapping\n Value of a: " << a << " | Value of b: " << b << std::endl << std::endl;
+
+
+	// Swapping the values using swapNum (number specific swapping)
```
Comment (2G-Afroz) [inline Algorithms/Utils/Swap/Swap2.cpp:27]: This line.

**Review by 2G-Afroz — COMMENTED** [review]
I think, there should be single file named ```Swap.cpp```. In this file, there can be multiple functions for swapping the values of different data ( of same data type ).
It will help to reduce redundancy of extra code.

**Comment by Alaz-Oz** [comment]
@2G-Afroz review it. It seems fine to me now.

**Review by 2G-Afroz — APPROVED** [review]

**Pull request merged**